upf/test/test_grpc_server.py

```diff
@@ -0,0 +1,53 @@
+from upf.test import TestCase
+from upf.shortcuts import *
+from upf.test.examples import get_example_problems
+
+
+class TestProtobufFactory(TestCase):
+    def setUp(self):
+        TestCase.setUp(self)
+        try:
+            import grpc
+        except ImportError:
+            self.skipTest("grpc module not found")
+        from upf.grpc.server import UpfGrpcServer
+        from upf.grpc.client import UpfGrpcClient
+
+        self.ps = get_example_problems()
+        self.server = UpfGrpcServer(30000)
+        self.server.start()
+        self.client = UpfGrpcClient("localhost", 30000)
+
+    def test_basic(self):
+        problem = self.ps['basic'].problem
+        plan_a = self.client(problem)
+        with OneshotPlanner(name='tamer', params={'weight': 0.8}) as planner:
```

Comment:
Maybe use a `Validator` instead of a `OneshotPlanner`?

Reply:
Does `Validator` create a plan? If there is a way to run tests without tamer it may be preferable.

Reply:
It can check if a plan is valid (and we have a pure-python plan validator). See here: https://github.com/aiplan4eu/upf/blob/master/upf/test/test_validator.py#L32